package/yast2-samba-server.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-samba-server
-Version:        3.1.16
+Version:        3.1.17
```

Comment:
Also for master current branch should use 3.2.X version number, I already merged a PR with `3.2.0` so just use `3.2.1`